config/default/kustomization.yaml

```diff
@@ -1,5 +1,3 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
```

Comment:
I'm not sure we should include these changes...  With more recent versions of kustomize bases is deprecated in favor of resources.

Reply:
This is consistent with new kubebuilder (2.1.x) generated yaml

Reply:
Dropping the apiVersion and kind, that is.

Reply:
Right, but it isn't necessarily consistent with the latest kustomize docs. I'm wondering if we should work on getting kubebuilder updated for this instead.

Reply:
I removed these because I was using cluster-api-provider-aws as a reference which didn't include them. Maybe we can create an issue on kubebuilder to track this.